src/cbc_sdk/platform/legacy_alerts.py

```diff
@@ -28,19 +28,22 @@ class LegacyAlertSearchQueryCriterionMixin(CriteriaBuilderSupportMixin):
 
     def set_categories(self, categories):
         """
-        Restricts the alerts that this query is performed on to the specified categories.
+        The field `categories` was deprecated and not included in v7.  This method has been removed.
+
+        In Alerts v7, only records with the type THREAT are returned.
+        Records that in v6 had the category MONITORED (Observed) are now Observations
+        See [Developer Network Alerts v6 Migration](https://developer.carbonblack.com/reference/carbon-black-cloud/guides/api-migration/alerts-migration/)
+        for more details.
 
         Args:
             categories (list): List of categories to be restricted to. Valid categories are
                                "THREAT", "MONITORED", "INFO", "MINOR", "SERIOUS", and "CRITICAL."
 
         Returns:
```

Comment:
This should be `Raises` instead of `Returns` now

Reply:
Done